modules/virt-golden-images-namespace-cli.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-rh-images-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-golden-images-namespace-cli_{context}"]
+= Configuring a custom namespace for golden images by using the CLI
+
```

Comment:
I am unsure about the wording.
Isn't "system-defined boot sources" the word which is used in the documentation instead of golden images?
Golden images is technically correct, and used in other places like https://www.redhat.com/en/topics/linux/what-is-a-golden-image , I just wonder if the wording is consistent here.

If the term "system-defined boot sources" would be used, the hint about "The customized namespace is ignored for modified golden images."  might be not relevant anymore.

Reply:
@dominikholler Great point about consistency! You're right that we use "system-defined boot sources" here: https://docs.openshift.com/container-platform/4.17/virt/storage/virt-automatic-bootsource-updates.html 

IIRC, long ago, we avoided the phrase "golden images," but a few releases ago, we switched to using the term because it had become an industry standard. However, the "Managing automatic boot source updates" page does not use the phrase "golden images" at all. 

I believe @fabiand was heavily involved in these discussions. Fabian, do you have any input on how these phrases are used in the downstream docs? Thanks!

Reply:
just wondering, should we document the behaviour of custom/modified dataImportCronTemplate?
Because its a bit different
@dominikholler

Reply:
Consistency within the page, and consistency accross the docs.

Thus if we overall move to golden images, thne let's do it here as well.
Thus either we updat ethe whole page to use golden images - or we stick to "system-provided boot sources" in order to kepe the consistency in this page.

my 2cts ..

Reply:
> Consistency within the page, and consistency accross the docs.
> 
> Thus if we overall move to golden images, thne let's do it here as well. Thus either we updat ethe whole page to use golden images - or we stick to "system-provided boot sources" in order to kepe the consistency in this page.
> 
> my 2cts ..

This PR does use "golden images" already, so I will keep the current wording. But we might want to update the other page that I linked ([automatic boot source updates](https://docs.openshift.com/container-platform/4.17/virt/storage/virt-automatic-bootsource-updates.html)) to use golden images.

Reply:
Please note that the wording of the documentation should be consistent to the UI in the long run.
An epic with stories for UX, UI and Doc teams could help to coordinate the works across the three teams.

Reply:
I opened the following bug for the terminology issue, because it's not within the scope of this PR. https://issues.redhat.com/browse/CNV-53972